content/docs/for-buildpack-authors/how-to/write-buildpacks/specify-env.md

```diff
@@ -3,15 +3,108 @@ title="Specify the environment"
 weight=99
 +++
 
+Environment variables are a common way to configure various buildpacks at build-time.
+
 <!--more-->
 
-This page is a stub! The CNB project is applying to [Google Season of Docs](https://developers.google.com/season-of-docs/docs/timeline) to receive support for improving our documentation. Please check back soon.
+### POSIX Path Variables
+
+The following layer path environment variables MUST be set by the `lifecycle` during the `build` and `launch` phases in order to make buildpack dependencies accessible.
```

Comment:
I'm gonna do a more detailed pass over this, but at first glance it seems that the text (especially where it pulls from the spec) is geared more toward a lifecycle implementor vs a buildpack author. I think we want to see language like, "to add a directory to the $PATH variable for buildpacks that follow, create a `<layers>/bin` directory and place any executables that you want to make available within that directory. The lifecycle will automatically add the directory to the $PATH for buildpacks that follow."

Reply:
@natalieparellano any idea when you'll have a detailed look at this? Thanks